### PR TITLE
Fixes #2490 add compare results via context menu

### LINKED ISSUE #2490: Compare Results: add new comparison via the context menu
Add 2 context menu items to every comparison folder and to the root "Comparisons" node:
* "Add compare results: Individual simulations"
* "Add compare results: Population simulations"

grafik grafik

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.Assets/PKSimConstants.cs [hunk 1]
```diff
@@ -1203,7 +1203,9 @@ public static class MenuNames
          public static readonly string GeneratePretermsData = "Generate Preterms Data";
          public static readonly string Comparison = "Compare Results";
          public static readonly string IndividualSimulationComparison = "Individual Simulations";
-         public static readonly string PopulationPopulationComparison = "Population Simulations";
+         public static readonly string PopulationSimulationComparison = "Population Simulations";
+         public static readonly string AddIndividualSimulationComparison = $"Add {IndividualSimulationComparison.ToLowerInvariant()} comparison";
```
Comment (msevestre) [inline src/PKSim.Assets/PKSimConstants.cs:1207]: @Yuri05 I think it reads better but happy to change to suggested text

**Comment by msevestre** [comment]
well I am going to add the other one after all

**Comment by msevestre** [comment]
@Yuri05 @rwmcintosh This is still using the local core but I want to show the code.
This allows to add Simulation, Comparison but also PI and SA to subfolders!!

File: src/PKSim.Presentation/Presenters/Main/SimulationExplorerPresenter.cs [hunk 2]
```diff
@@ -316,6 +321,32 @@ private static bool shouldIgnoreDoubleClick(object tag)
          return tag.IsAnImplementationOf<UsedBuildingBlock>() || tag.IsAnImplementationOf<UsedObservedData>();
       }
 
+      public override IEnumerable<IMenuBarItem> AllCustomMenuItemsFor(ClassificationNode classificationNode)
+      {
```
Comment (msevestre) [inline src/PKSim.Presentation/Presenters/Main/SimulationExplorerPresenter.cs:325]: This is it: 
On custom items we

1. Save the localization of the classification
2. Add the custom menus

It was easy to add for PU and SA even it it was not specified.

**Review by msevestre — COMMENTED** [review]
Using the wrong core dll .
But you can see the code. Once core is merged, I can update and we can merge PKSim
So please review as if it was ready

**Review by Yuri05 — APPROVED** [review]

**Comment by msevestre** [comment]
@Yuri05 @rwmcintosh I have updated core. Going to merge

**Pull request merged**